data/human/transport missions.txt

```diff
@@ -3582,3 +3582,732 @@ mission "Hauler VI cargo prototype 3"
 			`The Testament enters the planet's atmosphere at high speed and lands somewhat uncontrolled near a waste processing plant. The cargo hold is glowing red hot and some areas in the cargo hold appear to be on fire.`
 			`	Veronica disembarks, walks over to you, and apologizes for the difficult trip. She explains that the cooling systems failed, causing the sushi calamity you experienced during the multiple legs of your journey.`
 			`	You are paid <payment> and she asks you not to talk about this escort job to anybody. You leave the docks quickly in order to escape the horrible smell of overheated fish that is coming from Veronica and her ship.`
+
+
+
+
+	name "Take family home to <planet>"
+	description "You meet a family worried about the potential destruction of war in the Free Worlds and want to cut their vacation short and go home to <destination>."
```

Comment:
description "A family worried about the potential destruction in the Free Worlds wants to cut their vacation short and go home to <destination>."